### PR TITLE
Switch from setuphelper to setuptools.

### LINKED ISSUE #69: Switch from epsilon.setuphelper to setuptools
I believe all of Imaginary's dependencies have already switched, so this is the last step to making Imaginary `pip install`able in the normal way.

### PR DESCRIPTION
**Pull request by mithrandi**
Fixes #69.

### CODE REVIEW

**Comment by glyph** [comment]
Coverage is here: https://coveralls.io/builds/3926479

File: setup.py [hunk 1]
```diff
@@ -33,22 +33,14 @@
...
-
-        # Likewise.
-        "pytz", "pyasn1>=0.1.7",
-
-        # Likewise.  PyCrypto has a history of breaking Conch in new releases
-        # so pin it.
+        # PyCrypto has a history of breaking Conch in new releases so pin it.
         "PyCrypto==2.6",
 
         "mantissa>=0.7.0",
```
Comment (glyph) [inline setup.py]: If we're going to depend on Mantissa's dependencies being correct, shouldn't we depend on a version of mantissa whose dependencies are correct?

Comment (mithrandi) [inline setup.py]: I bumped the Mantissa dependency appropriately, as well as removing the PyCrypto dependency (I think it's okay to trust Mantissa to depend on the right version of that, too, if necessary). I can revert the latter change if you like.

Comment (glyph) [inline setup.py]: Nope, that sounds good.

**Pull request merged**